### DISCUSSION TITLE
Should Input/Output nodes be first/second children instead of first/last?

### DISCUSSION THREAD

**Discussion by cqc-alec** [Quantinuum/hugr#121] [Specification]
This would make it slightly easier to add new non-IO children.

**Comment by ss2165** [comment]
@acl-cqc Do you have any objections? I think this is a good idea

**Reply by aborgna-q** [reply]
👍

**Comment by acl-cqc** [comment]
Not really. I think we could do with a utility method like your `boundary` that returns a pair/array<2> of Nodes given the DFG container, but given that, fine

**Comment by ss2165** [comment]
Issue #146 
PR #148
